Production/src/TauJet.cc

```diff
@@ -334,7 +360,8 @@ void TauJetBuilder::Build()
 
         for(size_t pfCandIndex = 0; pfCandIndex < cands_.size(); ++pfCandIndex) {
             const auto& pfCand = cands_.at(pfCandIndex);
-            if(!hasMatch(pfCand.polarP4())) continue;
+	    auto SVidx = collectSV(pfCand);
+            if(!hasMatch(pfCand.polarP4()) && SVidx<0) continue;
```

Comment:
Unless I'm mistaken, this condition implies that only PF candidates from SVs will be stored in `tauJet`, which isn't really what we want. So `SVidx < 0` should be removed.

Reply:
This condition excludes PF candidates which are not in the cone and with no associated SV. However it keeps PF candidates inside and outside the cone whether they have a SV or not so previously stored candidates are still here.

Reply:
Ah right, sorry, I misunderstood the logic. Then it's fine👍🏻